src/index.ts

```diff
@@ -66,18 +64,23 @@ class LocalizeAssetsPlugin<LocalizedData = string> {
 		this.localizeCompiler = (
 			this.options.localizeCompiler
 				? this.options.localizeCompiler
-				: function (localizerArguments) {
-					const [key] = localizerArguments;
-
-					if (localizerArguments.length > 1) {
-						this.emitWarning(`[${name}] Ignoring confusing usage of localization function: ${functionName}(${localizerArguments.join(', ')})`);
-						return key;
-					}
+				: {
+					[this.options.functionName ?? '__'](localizerArguments) {
+						const [key] = localizerArguments;
+
+						if (localizerArguments.length > 1) {
+							const code = stringifyAst(this.callNode);
+							this.emitWarning(`[${name}] Ignoring confusing usage of localization function: ${code})`);
```

Comment:
I'm worried this code part could potentially be very long/multi-line. Maybe we can strip new lines/indents and truncate it after 100 characters or something?

Reply:
Good idea, thanks

Reply:
2dafc7a

Reply:
`stringifyAst` already generates code with no whitespace, by the way: https://github.com/privatenumber/webpack-localize-assets-plugin/blob/develop/src/utils/stringify-ast.ts#L4